packages/arb-avm-cpp/avm_values/include/avm_values/code.hpp

```diff
@@ -135,25 +152,116 @@ struct CodeSnapshot {
 };
```

Comment:
This isn't unique to this PR, but `op_count` on the line above in `CodeSnapshot` should actually be called `next_segment_num` as that's its value.

Reply:
Done